packages/atomic/CONTRIBUTING.md

```diff
@@ -0,0 +1,255 @@
+  </atomic-commerce-layout>
+</atomic-commerce-interface>
+```
+
+</AtomicDocTemplate>
+```
+
+**Required content within the template for the Usage section:**
+- Component description and purpose
+- Basic usage examples with proper HTML structure
+- Any important notes or constraints
+- Integration context within the atomic interface
+
+### Unit Tests (.spec.ts)
+
+Comprehensive unit testing for component logic and behavior.
+
+**Testing principles:**
+- Follow the [Atomic test conventions](https://coveord.atlassian.net/wiki/spaces/JSUI/pages/5221122259/Atomic+test+conventions)
+- Wrap all tests under a single `describe` with the component name
+- Use `it` function (not `test`) for individual test cases
+- Start test descriptions with "should"
+
+**Required test coverage:**
+- **Test all possible functionalities** - Every prop and public method of the component must be tested
+- **Test all condition paths** - Cover all if/else branches, switch cases, and conditional logic
+- **Test all DOM attributes** - Validate proper rendering of ARIA labels, CSS parts, and other DOM properties
+- **Test component states** - Verify component behavior in different states (loading, error, success, etc.)
+- **Test user interactions** - Cover all click handlers, keyboard navigation, and event listeners
+- **Test property/attribute binding** - Ensure all component properties correctly affect the rendered output
+
+
+### End-to-End Tests (e2e/)
+
+Integration testing that validates component behavior in real browser environments.
+
+**Files in e2e/ directory:**
+- `atomic-component-name.e2e.ts`: Main E2E test file
+- `fixture.ts`: Test setup and configuration
+- `page-object.ts`: Abstraction layer for component interactions
+
+**Purpose:**
+- Follow the [Atomic test conventions](https://coveord.atlassian.net/wiki/spaces/JSUI/pages/5221122259/Atomic+test+conventions)
+- **Test the golden path** - Focus on the most common, successful user journey through the component
+- **Test accessibility compliance** - Validate ARIA attributes, keyboard navigation, and screen reader compatibility
+- **Keep it simple** - Usually should only include the default story and test a very simple golden path
+
+**Example:** See [`atomic-product-children.e2e.ts`](src/components/commerce/atomic-product-children/e2e/atomic-product-children.e2e.ts) for a reference implementation.
+
+### Styles (.tw.css)
+
+Component styling using Tailwind CSS utility classes.
+
+> To use Tailwind CSS utility classes in your component, you must add the `@withTailwindStyles` decorator to your component class
+
+**When to use:**
+- **Styling is usually contained in the component's `styles` property** - Most components can define all their styles directly within the component using Lit's static styles
+- **Use .tw.css files only when importing external styles** - Create a separate .tw.css file only when you need to import styles from another file or when styles are too complex to maintain inline
```

Comment:
An example of using an external stylesheet would be useful here.

I didn't include one in the component generation template because it would bloat it too much.